shell/key-bindings.bash

```diff
@@ -17,8 +17,11 @@
 # Key bindings
 # ------------
 __fzf_select__() {
-  local cmd opts
-  cmd="${FZF_CTRL_T_COMMAND:-"command find -L . -mindepth 1 \\( -path '*/.*' -o -fstype 'sysfs' -o -fstype 'devfs' -o -fstype 'devtmpfs' -o -fstype 'proc' \\) -prune \
+  local cmd opts macos_exclude
+  if [[ "$(uname -s)" == "Darwin" ]]; then
+    macos_exclude="-o -xattrname 'com.apple.containermanager.uuid'"
+  fi
+  cmd="${FZF_CTRL_T_COMMAND:-"command find -L . -mindepth 1 \\( -path '*/.*' -o -fstype 'sysfs' -o -fstype 'devfs' -o -fstype 'devtmpfs' -o -fstype 'proc' $macos_exclude \\) -prune \
```

Comment:
How about if we keep it simple and just put `-xattrname 'com.apple.containermanager.uuid'` regardless of the platform? Will there be any negative consequences for users?

Reply:
Unfortunately yes. When find encounters a predicate it doesn't recognize it bails immediately and produces no results. Which actually brings up another issue: if someone on macOS has installed GNU find and deliberately placed it earlier on their PATH this command will similarly fail. On the one hand doing this is a "buyer beware" sort of situation but it's not _that_ uncommon.

I can think of a few ways of dealing with this:
1. Ignore it. Buyer beware.
2. Instead of `uname` check if `find /dev/null -xattrname 'com.apple.containermanager.uuid' 2> /dev/null` succeeds.
3. Instead of `command find` use `/usr/bin/find`.

2 is probably the least disruptive but is still adding complexity.

Reply:
Oh I see, `-xattrname` is not available on GNU find. It's unfortunate.

Reply:
I went ahead and did number 2. I haven't been able to test this on fish yet but otherwise I think this should work.

Reply:
Thanks. Honestly, I don't know how I feel about this. Are these conditional branches which add complexity and degrade the readability worth it?

Reply:
Yeah, I'm no longer sure. The fact that my most recent commit had a bug is a vote against it. I'll keep it open for a bit longer though.